packages/frontend/package.json

```diff
@@ -12,6 +12,7 @@
     "@sentry/browser": "^6.4.1",
     "bip39-light": "^1.0.7",
     "connected-react-router": "^6.6.0",
+    "dataloader": "^2.1.0",
```

Comment:
did we need to introduce this new package, how were we handling fetch requests before?

Reply:
this new package is specifically to cache and batch on the frontend any fetch requests that we make. this dependency was originally in near-contract-helper, because it gave us caching and batching/coalescing requests out of the box, so we're just moving it to the frontend.